patches/node/build_add_gn_build_files.patch

```diff
@@ -1176,17 +1163,17 @@ index 0000000000000000000000000000000000000000..2e07ea28437ee25ddaea4d730e1a65a5
 +    "lib/internal/streams/state.js",
 +    "lib/internal/streams/pipeline.js",
 +    "lib/internal/streams/end-of-stream.js",
-+    "//v8/tools/splaytree.js",
-+    "//v8/tools/codemap.js",
-+    "//v8/tools/consarray.js",
-+    "//v8/tools/csvparser.js",
-+    "//v8/tools/profile.js",
-+    "//v8/tools/profile_view.js",
-+    "//v8/tools/logreader.js",
-+    "//v8/tools/arguments.js",
-+    "//v8/tools/tickprocessor.js",
-+    "//v8/tools/SourceMap.js",
-+    "//v8/tools/tickprocessor-driver.js",
++    "deps/v8/tools/splaytree.js",
++    "deps/v8/tools/codemap.js",
++    "deps/v8/tools/consarray.js",
++    "deps/v8/tools/csvparser.js",
++    "deps/v8/tools/profile.js",
++    "deps/v8/tools/profile_view.js",
++    "deps/v8/tools/logreader.js",
++    "deps/v8/tools/arguments.js",
++    "deps/v8/tools/tickprocessor.js",
++    "deps/v8/tools/SourceMap.js",
++    "deps/v8/tools/tickprocessor-driver.js",
```

Comment:
hmm, we shouldn't be using the code in `deps/v8`. can we change this back to `//v8`?

Reply:
V8 changed this here: https://github.com/v8/v8/commit/56bf834a2b122616d00c6cc85243e4d80a1ca241 which is why this occurred but i agree - we should probably update our filename generation logic accordingly 🤔